### ISSUE TITLE
Allow image/video attachments to be taken directly from camera

### ISSUE THREAD

**Issue by fjwong** [sujrd/faiadashu#15]

**Comment by hanny-ph** [comment]
@fjwong please correct me if I'm wrong, basically before opening either `file_selector`, camera, or gallery, we need to show a dialog of the available options, for example:

![options](https://github.com/sujrd/faiadashu/assets/20153323/6ac162ce-4d0f-438b-b146-036f658788a4)

Sorry, in the image, the dialog uses an old design, but I hope it represents the idea.

**Comment by fjwong** [comment]
@hanny-ph Yes, something like that is more or less what we had in mind. :)
